gulpfile.js

```diff
@@ -47,6 +48,9 @@ function themeName() {
 	return process.env.SC_THEME || 'default';
 }
 
+const EXIT_ON_ERROR = process.env.GULP_EXIT_ON_ERROR === 'true'
```

Comment:
how to set this in build and test? i would use true as default for `GULP_EXIT_ON_ERROR `

Reply:
but if true is the default value, all devs need to set the env during development. (the soft fail was introduced for dev to not be required to restart gulp after every error)

Reply:
GULP_EXIT_ON_ERROR || env !== develop